test/Nancy.Tests/Resources/Link.Texts.Designer.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Nancy.Tests.Resources {
     using System;
-    
-    
+    using Nancy.Extensions;

```

Comment:
I am absolutely sure that editing the Designer files was a bad idea, but I didn't have any good ones. I'm all ears.